sandpiper/processes/wps_evaluate_rule.py

```diff
@@ -69,27 +74,27 @@ def __init__(self):
         )
 
     def _handler(self, request, response):
-        rule, parse_tree_path, variables_path, loglevel = [
-            arg[0] for arg in collect_args(request, self.workdir).values()
-        ]
+        rules, parse_tree_path, variables_path, loglevel = collect_args(
+            request, self.workdir
+        ).values()
 
         log_handler(
             self,
             response,
             "Starting Process",
             logger,
-            log_level=loglevel,
+            log_level=loglevel[0],
```

Comment:
We should process this variable a bit more so that we do not have to use the index. Same goes for the `parse_tree_path`.